hack/verify-yamllint.sh

```diff
@@ -37,8 +37,8 @@ if [ $# != 0 ]; then
 fi
 
 if ! command -v yamllint >/dev/null 2>&1; then
-  echo >/dev/stderr "ERROR: yamllint not found - please install with: ${pip} install -r ${pip_requirements}"
-  exit 1
+  echo "yamllint not found - installing with: ${pip} install -r ${pip_requirements}"
+  ${pip} install -r ${pip_requirements}
```

Comment:
It would be great to have a comment saying we assume `pip3` is already installed.

Reply:
Fair point, I think we can address in followup. Other scripts in this repo assume the presence of python3, which implies pip3. Not sure if they verify their assumptions either